src/main/java/com/mixpanel/android/viewcrawler/ViewCrawler.java

```diff
@@ -725,24 +720,24 @@ private void handleEditorBindingsCleared(JSONObject clearMessage) {
                 Log.e(LOGTAG, "Bad clear request received", e);
             }
 
-            updateEditState();
+            applyVariantsAndEventBindings();
         }
 
         private void handleEditorTweaksReceived(JSONObject tweaksMessage) {
             try {
+                mEditorTweaks.clear();
```

Comment:
I'm a bit curious about this.. Why do we clear mEditorTweaks and then add all the tweaks to it, while in handleEditorChangeReceived we only do mEditorChanges.put?